ballerina/tests/file-test.bal

```diff
@@ -77,6 +78,19 @@ function testCopyDir() {
     }
 }
 
+@test:Config {}
+function testCopyDir2() returns error? {
+    string targetPath = "tests/resources/temp-dir/nested-file.txt";
+    error? copyResult = copy(srcDir, "tests/resources/temp-dir", REPLACE_EXISTING);
+    if copyResult is error {
+        test:assertFail("Directory not copied!");
+    } else {
+        string readContent = check io:fileReadString(targetPath);
+        test:assertEquals(readContent, "Hi");
+        check io:fileWriteString(targetPath, "");
+    }
+}
+
```

Comment:
We need to have a test case where the target directory doesn't exist

Reply:
That is already existing:https://github.com/ballerina-platform/module-ballerina-file/blob/master/ballerina/tests/file-test.bal#L73

Reply:
Then how come it is passing. is the problem only when we try to replace existing files.